### PR TITLE
Card component document

### PR DESCRIPTION
**Pull request by bariscanyilmaz**
### Description

Create card component document.

### Checklist

- [x] I fully tested it as developer / designer and created unit / integration tests
- [x] I documented it (or no need to document or I will create a separate documentation issue)

### CODE REVIEW

**Review by braim23 — APPROVED** [review]

File: docs/en/UI/Angular/Card-Component.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```ts
+// my-feature.module.ts
+
+import { ThemeSharedModule } from '@abp/ng.theme.shared';
+import { CardDemoComponent } from './chart-demo.component';
+
+@NgModule({
+  imports: [
+    SharedModule,
```
Comment (rqx110) [inline docs/en/UI/Angular/Card-Component.md]:
```suggestion
    ThemeSharedModule ,
```

File: docs/en/UI/Angular/Card-Component.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Card Component
+
+The ABP Card Component is a wrapper component for the Bootstrap card class.
+
+## Usage
+
+ABP Card Component is a built-in `ThemeSharedModule` component. If you've imported a shared module into your feature module, you don't need to do anything.
```
Comment (hikalkan) [inline docs/en/UI/Angular/Card-Component.md]:
```suggestion
ABP Card Component is a part of the `ThemeSharedModule` module. If you've imported that module into your module, you don't need to import it again. If not, first import it as shown below:
```

File: docs/en/UI/Angular/Card-Component.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-card-demo',
+  template: ` 
+    <abp-card [cardStyle]="{width: '18rem'}">
+      <abp-card-body>
+        <abp-card-title>Lorem Ipsum</abp-card-title>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla commodo condimentum ligula, sed varius nibh eleifend sit amet. Maecenas facilisis vel arcu nec maximus.</abp-card-body>
```
Comment (hikalkan) [inline docs/en/UI/Angular/Card-Component.md]:
```suggestion
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla commodo condimentum ligula, sed varius nibh eleifend sit amet. Maecenas facilisis vel arcu nec maximus.
      </abp-card-body>
```

**Review by hikalkan — CHANGES_REQUESTED** [review]

**Comment by hikalkan** [comment]
We show usages of the `abp-card`, `abp-card-body` and `abp-card-title` elements. Isn't there more? If so, we should write a list of all defined elements and all of their options.

**Comment by bariscanyilmaz** [comment]
> We show usages of the `abp-card`, `abp-card-body` and `abp-card-title` elements. Isn't there more? If so, we should write a list of all defined elements and all of their options.

We haven't implemented all elements yet. As new elements are added, document will be updated.

**Review by mahmut-gundogdu — APPROVED** [review]

**Review by hikalkan — APPROVED** [review]

**Pull request merged**